app/uk/gov/hmrc/fhregistrationfrontend/forms/mappings/Mappings.scala

```diff
@@ -81,6 +81,10 @@ object Mappings {
   def unincorporatedBodyName =
     nonEmptyText verifying Constraints.pattern(
       "^[a-zA-Z0-9\u00C0-\u00FF !#$%&'‘’\"“”«»()*+,./:;=?@\\[\\]|~£€¥\\u005C\u2014\u2013\u2010\u005F\u005E\u0060\u002d]{1,120}$".r)
+
+  def partnerhshipName =
+    nonEmptyText verifying Constraints.pattern(
+      "^[a-zA-Z0-9\u00C0-\u00FF !#$%&'‘’\"“”«»()*+,./:;=?@\\[\\]|~£€¥\\u005C\u2014\u2013\u2010\u005F\u005E\u0060\u002d]{1,120}$".r)
```

Comment:
This accepts 1 - 120 chars but the error message states partnership name should be 140 chars or less